README.md

```diff
@@ -1,22 +1,336 @@
-# Diy Json Serializer
+# DIY JSON Serializer
 
 ## Objectives
 
-  1. Understand what JSON is
-  2. Create and Return simple JSON Objects
-  3. Load those objects with AJAX into HTML
-
-## Outline
-
-  * Grab the code from previous readme.
-  * Goal here is to re-do the post show page to be data backed by JS
-  * Previously, we just dumped data out at different routes. This isn't sustainable. We need a way to hit just one route, and get structured data back.
-  * Traditionally this was solved with HTML. HTML has both data and formatting, not good
-  * Just want pure data, used to be given via XML. XML was annoying
-  * Then came JSON
-  * Show them what JSON looks like. Just a dictionary or array of dictionaries!
-  * JSON is a way of "serializing" data. Remember we need to send everything as strings. Serialization means convert from the ruby format of an object, to a string format of an object. 
-  * Let's write our own JSON post serializer.
-  * Make it a separate class with a single class method that takes in the post object and returns a string of JSON
-  * don't use `to_json` or anything like that. Have them by hand build up a JSON dictionary of the important bits of the post
-  * The show page, make it backed by JSON.
+  1. Understand what JSON is.
+  2. Create and Return simple JSON Objects by hand.
+  3. Load those objects with AJAX into HTML.
+
+## Lesson
+
+Continuing with our blog application, we'll be improving our blog post
+API to provide more structure and more efficient data access. Make sure
+to run `rake db:seed` to set up some starting data!
+
+Last time, we created an endpoint for the single piece of data we wanted
+to access. That worked well enough, but doesn't scale. If we want to use
+AJAX to get all the data from a blog post, we'd be creating so many
+endpoints, and making so many GET requests.
+
+What we really need is one endpoint to make one request and get back
+structured data representing our blog post.
```

Comment:
That's fair -- I'm erring on the side of being too explicit, but I think you're right that it can come across as too spoon-feedy.

I guess what I'm trying to guard against is the tendency (that I've noticed in myself) to use these ostensibly intuitive terms that might actual deserve some unpacking (e.g., "structured data," "implementation detail," etc.).

But anyway I think you make a valid point.